src/Scene/LayersConfiguration.js

```diff
@@ -18,6 +18,12 @@ function LayersConfiguration() {
     this.geometryLayers = [];
     this.colorLayers = [];
     this.elevationLayers = [];
+    this.lightingLayers = [];
```

Comment:
Currently it *is* a globe-only feature.
But while I understand that we might want lighting on other scenario this doesn't mean it makes sense to store lighting as a layer.

If you don't like storing this in `Globe`, what about these 2 proposals:
  * store lighting position in `Scene` ?
  * store lighting position in imagery layers attributes (alongside `visible`, `frozen` attributes etc) ?